@here/harp-mapview/lib/Utils.ts

```diff
@@ -21,37 +21,19 @@ const MINIMUM_OBJECT3D_SIZE_ESTIMATION = 1000;
 
 const MINIMUM_ATTRIBUTE_SIZE_ESTIMATION = 56;
 
-export namespace MapViewUtils {
-    //Caching those for performance reasons.
-    const groundPlane = new THREE.Plane(new THREE.Vector3(0, 0, 1));
-    const groundSphere = new THREE.Sphere(undefined, EarthConstants.EQUATORIAL_RADIUS);
-    const cameraZPosition = new THREE.Vector3(0, 0, 0);
-    const rotationMatrix = new THREE.Matrix4();
-    const unprojectionMatrix = new THREE.Matrix4();
-    const rayCaster = new THREE.Raycaster();
-
-    /**
-     * Yaw rotation as quaternion. Declared as a const to avoid object re-creation in certain
-     * functions.
-     */
-    const yawQuaternion = new THREE.Quaternion();
-
-    /**
-     * Pitch rotation as quaternion. Declared as a const to avoid object re-creation in certain
-     * functions.
-     */
-    const pitchQuaternion = new THREE.Quaternion();
-
-    /**
-     * Cached quaternion for intermediate maths to avoid creating new instances.
-     */
-    const tmpQuaternion = new THREE.Quaternion();
-
-    /**
-     * Cached matrix for intermediate maths to avoid creating new instances.
-     */
-    const tmpMatrix = new THREE.Matrix4();
+//Caching those for performance reasons.
```

Comment:
Maybe stupid question, but why you have extracted helpers outside the namespace? Are they used somewhere else?

Reply:
Yes that was a request from @robertoraggi, because they should not be in the API of the Utils namespace, they are just there to avoid creating and dumping objects every frame.